jax/interpreters/partial_eval.py

```diff
@@ -74,7 +74,7 @@ def instantiate_const(self, tracer):
     if isinstance(pv, AbstractValue):
       return tracer
     elif pv is None:
-      if type(const) in core.literalable_types and onp.shape(const) == ():
```

Comment:
I am curious, what are non core.literalable_types. In other words, what things will still be lifted as consts?